### PR TITLE
#187419030 user should see recommended product when viewing single product

### PR DESCRIPTION
**Pull request by JacquelineTuyisenge**
## Description of Task to be completed?

Clone branch `ft-related-products-187419030` 

run `npm run dev` to test the feature

go to products, then single product
you should see *you may also like:* text and the list of related products

What are the relevant pivotal tracker/Trello stories?
#187419030

### CODE REVIEW

File: src/pages/RecommendedProduct.tsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+		}
+	}, [dispatch, prodId]);
+
+	return (
+		<>
+			<div className="text-2xl pb-2 mt-5">
+				<h2>You may also like:</h2>
+			</div>
+			<div className="grid mobile:grid-cols-2 gap-5 pb-4 tablet:grid-cols-3 ipad:grid-cols-3 laptop:grid-cols-4 desktop:grid-cols-4 h-full rounded-md">
+				{isLoading && <p>Loading...</p>}
```
Comment (GarrixA) [inline src/pages/RecommendedProduct.tsx]: @JacquelineTuyisenge Find a best loader and replace this kind of loading

File: src/pages/RecommendedProduct.tsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+	useEffect(() => {
+		if (prodId) {
+			dispatch(getRecommendedProducts(prodId)).unwrap();
+		}
+	}, [dispatch, prodId]);
+
+	return (
+		<>
+			<div className="text-2xl pb-2 mt-5">
+				<h2>You may also like:</h2>
```
Comment (GarrixA) [inline src/pages/RecommendedProduct.tsx]: This could be visible only when there is a recommended products

**Review by GarrixA — CHANGES_REQUESTED** [review]

File: src/pages/RecommendedProduct.tsx (hunk 1 above)
Comment (JacquelineTuyisenge) [reply src/pages/RecommendedProduct.tsx]: done

File: src/pages/RecommendedProduct.tsx (hunk 2 above)
Comment (JacquelineTuyisenge) [reply src/pages/RecommendedProduct.tsx]: done

**Review by GarrixA — APPROVED** [review]

**Review by kwizera-bonheur25 — APPROVED** [review]

**Review by leandreAlly — APPROVED** [review]

**Pull request merged**